src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -101,21 +106,31 @@ object HttpGoogleServicesDAO extends GoogleServicesDAO with FireCloudRequestBuil
     googleCredential.getAccessToken
   }
 
-  def getTrialBillingManagerAccessToken = {
-    val googleCredential = new GoogleCredential.Builder()
+  def getTrialBillingManagerCredential(impersonateUser: Option[String] = None): Credential = {
+    val builder = new GoogleCredential.Builder()
       .setTransport(httpTransport)
       .setJsonFactory(jsonFactory)
       .setServiceAccountId(trialBillingPemFileClientId)
-      .setServiceAccountScopes(authScopes ++ billingScope)
+      .setServiceAccountScopes(authScopes ++ billingScope :+ ComputeScopes.CLOUD_PLATFORM)
```

Comment:
The `CLOUD_PLATFORM` scope is not needed, unless we are able to make requests as the end user to get project information.